klarna-on-demand/src/test/java/com/klarna/ondemand/CryptoImplTest.java

```diff
@@ -0,0 +1,56 @@
+package com.klarna.ondemand;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.SignatureException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(RobolectricTestRunner.class)
+@Config(emulateSdk = 18)
+@PrepareForTest(CryptoImpl.class)
+@PowerMockIgnore({ "org.mockito.*", "org.robolectric.*", "android.*", "org.json.*" })
+public class CryptoImplTest {
+
+    private android.content.Context context;
+
+    @Before
+    public void init() {
+        context = Robolectric.application.getApplicationContext();
+    }
+
+    @Test
+    public void getPublicKeyBase64Str_shouldReturnSameKeyOnTwoConsecutiveCall () {
```

Comment:
1. Because we don't expose the private key (but the sign test make sure it's the same)
2. The method name is get**PublicKey**Base64Str_shouldReturnSameKeyOnTwoConsecutiveCall